README.md

```diff
@@ -15,7 +15,7 @@ There are two main types of audio datasets: speech datasets and audio event/musi
 * [Awesome_Diarization](https://github.com/jim-schwoebel/awesome-diarization) - A curated list of awesome Speaker Diarization papers, libraries, datasets, and other resources. 
 * [BAVED](https://www.kaggle.com/a13x10/basic-arabic-vocal-emotions-dataset) - 1935 recording by 61 speakers (45 male and 16 female).
 * [CaFE](https://www.gel.usherbrooke.ca/audio/cafe.htm) - 6 different sentences by 12 speakers (6 fmelaes + 6 males).
-* [Common Voice](https://voice.mozilla.org/) - Common Voice is Mozilla's initiative to help teach machines how real people speak. 12GB in size; spoken text based on text from a number of public domain sources like user-submitted blog posts, old books, movies, and other public speech corpora.
+* [Common Voice]([https://voice.mozilla.org/](https://commonvoice.mozilla.org/en/datasets)) - Common Voice is Mozilla's initiative to help teach machines how real people speak. 12GB in size; spoken text based on text from a number of public domain sources like user-submitted blog posts, old books, movies, and other public speech corpora.
```

Comment:
I believe this is not right. You have essentially created a link inside the URL part of another link.

My proposal:

`[Common Voice]`<s>([https://voice.mozilla.org/]</s>`(https://commonvoice.mozilla.org/en/datasets)`<s>)</s>

Reply:
Sorry! You are right!